docs/translations/api-docs/data-grid/data-grid.json

```diff
@@ -1,106 +1,738 @@
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "components": {
+      "description": "Overridable components.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "componentsProps": {
+      "description": "Overridable components props dynamically passed to the component at rendering.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "density": {
+      "description": "Set the density of the grid.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "disableColumnFilter": {
+      "description": "If <code>true</code>, column filters are disabled.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "disableColumnMenu": {
+      "description": "If <code>true</code>, the column menu is disabled.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "disableColumnSelector": {
+      "description": "If <code>true</code>, hiding/showing columns is disabled.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "disableDensitySelector": {
+      "description": "If <code>true</code>, the density selector is disabled.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "disableRowSelectionOnClick": {
+      "description": "If <code>true</code>, the selection on click on a row or cell is disabled.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "disableVirtualization": {
+      "description": "If <code>true</code>, the virtualization is disabled.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "editMode": {
+      "description": "Controls whether to use the cell or row editing.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "experimentalFeatures": {
+      "description": "Unstable features, breaking changes might be introduced. For each feature, if the flag is not explicitly set to <code>true</code>, the feature will be fully disabled and any property / method call will not have any effect.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "filterMode": {
+      "description": "Filtering can be processed on the server or client-side. Set it to &#39;server&#39; if you would like to handle filtering on the server-side.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "filterModel": {
+      "description": "Set the filter model of the grid.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {}
+    },
+    "getCellClassName": {
+      "description": "Function that applies CSS classes dynamically on cells.",
+      "notes": "",
+      "deprecated": "",
+      "typeDescriptions": {
+        "params": "With all properties from [[GridCellParams]].",
```

Comment:
Is it possible to have a link to the `GridCellParams` interface page, like we currently have?

https://github.com/mui/mui-x/blob/f238bc0c43515979401cda9e7500506589b7ce14/docs/translations/api-docs/data-grid/data-grid.json#L31

Reply:
Done in https://github.com/mui/mui-x/pull/9187/commits/25daa0569d925859d13b39142e3ec6b8b3a15620

I applied `linkify` only on the description. Now it's also applied to the notes and parameters descriptions